Add PxToRem package

xiyuesaves opened the pull request:
<!--
Your pull request will be reviewed automatically and by a human.

The manual review may take several days or weeks,
depending on the reviewer's availability and workload.
If you haven't received a comment on your pull request
and it wasn't merged either,
it just hasn't been reviewed yet.

---

Please ensure the automated reviews pass.
Follow the instructions provided, if necessary.
You can speed up the process
by [running some tests locally](https://packagecontrol.io/docs/submitting_a_package#Step_7).

You can request a review from @packagecontrol-bot
to manually trigger an automated review
if you don't need to push a new commit.
Do **NOT** open a new pull request!

In general, make sure you:

 1. Used `"tags": true` and not `"branch": "master"`
    (versioning docs: <https://packagecontrol.io/docs/submitting_a_package#Step_4>)
 2. Added a README to your repository so that users (and reviewers) 
    can understand what your package provides.

You should proceed with a short description of what the package does
and, in case one or multiple similar package already exists,
why you believe it is different and needed
below this line. -->

braver commented:
We already seem to have [some](https://packagecontrol.io/packages/rem-unit) [packages](https://packagecontrol.io/packages/REM%20PX) that provide this functionality. What does your package add to what's already out there?

xiyuesaves commented:
Fixed the bug that `rem-unit` could not convert negative numbers and added keywords to determine whether the plugin is enabled
![](/xiyuesaves/px-to-rem/raw/master/firstline.gif)

braver commented:
Well that sounds like you should open a PR on the existing package so it can be improved for all users of that package. Please try that first. And if that doesn’t work out for some reason, come back here and we can discuss some alternatives.
